modules/kernel/src/main/scala/mongo4cats/operations/Aggregate.scala

```diff
@@ -223,6 +224,20 @@ trait Aggregate extends AsJava {
     */
   def lookup(from: String, pipeline: Aggregate, as: String): Aggregate
 
+  /** Creates a \$addFields pipeline stage
+    *
+    * <p>With \$addFields, you can adds the new fields the document.</p>
+    *
+    * @param name
+    *   the name of the new field
+    * @param value
+    *   the value of the new field
+    * @return
+    *   the \$addFields pipeline stage [[https://www.mongodb.com/docs/manual/reference/operator/aggregation/addFields/]]
+    * @since 3.4
```

Comment:
I got the version from: https://www.mongodb.com/docs/v3.4/reference/operator/aggregation/addFields/